tests/rpmsigdig.at

```diff
@@ -1052,6 +1052,54 @@ echo $?
 gpgconf --kill gpg-agent
 RPMTEST_CLEANUP
 
+# Signing test(s) using Sequoia
+AT_SETUP([rpmsign --addsign sequoia])
+AT_KEYWORDS([rpmsign signature])
+AT_SKIP_IF([test x$PGP = xdummy])
+RPMDB_INIT
+
+RPMTEST_CHECK([
+# XXX This gpg2 import is unrelated to the actual signing, but unless gpg2
+# has run once, sq import fails due to some gpg-agent communication error.
```

Comment:
does it actually fail and exit with non-zero exit code? From my experiments, the agent failures were there, but the signatures went through just ok.

Reply:
Right, it actually does succeed if I just ignore stderr output. gpgconf fails noisily with "blah blah exit status 1" there but it doesn't indeed affect sq exit itself.

I'd rather not ignore stderr but then the gpg import is so ugly that ignoring stderr is probably the lesser evil here. Thanks for the tip.